app/src/main/java/com/github/orkest/View/profile/ProfileTopInterface.kt

```diff
@@ -123,6 +138,27 @@ fun ProfileTopInterface(viewModel: ProfileViewModel, scaffoldState: ScaffoldStat
     }
 }
 
+/**
+ * Send a Notification for the user when signing out
+ * This is an example to later finish developing the notifications
+ */
+fun sendNotification(context: Context, title: String, message: String, channelId: String) {
+    val notificationManager = context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+
+    val channel = NotificationChannel(channelId, "Sign Out Channel", NotificationManager.IMPORTANCE_DEFAULT)
+    notificationManager.createNotificationChannel(channel)
+
+    val notificationBuilder = NotificationCompat.Builder(context, channelId)
+        .setContentTitle(title)
+        .setContentText(message)
+        .setSmallIcon(android.R.drawable.ic_dialog_info)
+
+    notificationManager.notify(0, notificationBuilder.build())
+

```

Comment:
The notification ID is hard-coded to 0. This means that if multiple notifications are sent with the same ID, they will overwrite each other. It would be better to generate a unique notification ID for each notification.

Reply:
Ok, I've put a random generator now

Reply:
Actually, the most important thing is that it should be **unique**. Putting a random generator could still generate the same number. With a small number of notifications, it shouldn't be noticeable but it could still lead to unexpected behavior. So I would actually suggest a general number value accessible everywhere in the app and increment it everytime you use it as a notification ID. That way you are sure to have a unique ID and no notification will be overwritten.

Reply:
the overwrite isn't a problem, it will just notify of the latest update so I think we should keep it as it was